using_images/s2i_images/ruby.adoc

```diff
@@ -104,9 +104,11 @@ In order to dynamically pick up changes made in your application source code, yo
 
 *For Ruby on Rails applications*
 
-Run the built Rails image with the `RAILS_ENV=development` environment variable passed to the link:http://docker.io[Docker] `-e` run flag:
+Run the built Rails application with the `RAILS_ENV=development` environment
+variable passed to the running Pod. For an exisiting Deployment Config, you can
```

Comment:
Replace the `Deployment Config` with one of :
- \`\*DeploymentConfig\*\`
- deployment config